pref_sources/RubyMine/keymaps/Mavenlink Synced.xml

```diff
@@ -32,6 +77,29 @@
     <keyboard-shortcut first-keystroke="shift ctrl r" />
     <keyboard-shortcut first-keystroke="shift meta f8" />
   </action>
+  <action id="SelectNextOccurrence">
+    <keyboard-shortcut first-keystroke="ctrl g" />
+    <keyboard-shortcut first-keystroke="shift meta d" />
```

Comment:
I think we should discuss this further as a small group. I'm going to merge this in for now and we can swap them if agreed upon.